CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## master
 
+### Features
+
+* `[jest-each]` Move jest-each into core Jest
```

Comment:
```
Move `jest-each` into Jest core
```
?

Reply:
Also I think this is more of a chore action, cc @SimenB

Reply:
I agree - the feature has already landed in Jest, moving this package in is just a step on the way for circus to get the feature as well